src/interface.js

```diff
@@ -18,7 +18,7 @@ export default function(Target) {
         err.originalStack = err.stack;
         const firstLine = err.stack.split('\n')[0];
         this._asyncStack.unshift(firstLine);
-        // put the fake more helpful "async" stack on the thrown error
+        // put the fake more helpful "async" stack on the throw error
```

Comment:
are you sure that this correction is needed? "thrown error" sounds to me like "error that was thrown" which seems right.